application-ckeditor-ui/src/main/resources/CKEditor/InlineEditor.xml

```diff
@@ -161,6 +161,7 @@
       'data-sourceDocumentReference': XWiki.Model.serialize(config.document.documentReference),
       'data-sourceDocumentSyntax': config.document.syntax,
       'data-officeImporterSupported': config.enableOfficeImport,
+      'data-temporary-attachment-supported': config.enableTemporaryAttachmentUpload === true,
```

Comment:
I don't think this is needed anymore now that the config is done in ``ConfigSheet``.